src/app/helpers/link.js

```diff
@@ -2,7 +2,7 @@ import settings from 'settings';
 
 const EXTERNAL = /^((f|ht)tps?:)?\/\//;
 const ABSOLUTE_OPENSTAX = new RegExp(
-    `(https?://openstax.org|${window.location.origin})(?!/accounts|/oxauth|/blog-feed|/support)`
+    `(https?://openstax.org|${window.location.origin})/(?!accounts|oxauth|blog-feed|support)`
```

Comment:
@RoyEJohnson - will this work on dev and QA with https://openstax.org hardcoded in?

Reply:
Yes, it's openstax.org _or_ window.location.origin. It accounts for some links that are to openstax.org in the CMS data. The usual case is window.location.origin.